core/templates/pages/learner-dashboard-page/modal-templates/remove-activity-from-learner-dashboard-modal.controller.spec.ts

```diff
@@ -171,12 +176,13 @@ describe('Remove Activity From Learner Dashboard Modal Controller',
         });
       }));
 
-      it('should remove activity', function() {
-        expect(function() {
-          $scope.remove();
-        }).toThrowError('Section name is not valid.');
+      it('should remove activity from backend when clicking remove button',
```

Comment:
Hmmm, this seems to be a failure test. Should the name be different?

Reply:
Done.